doc/articles/composition.md

```diff
@@ -35,6 +35,102 @@ This thread will also be used for [independent animations](https://docs.microsof
 
 _There are a few known issues associated with the used of the compositor thread, [make sure to read the section below](#known-issues)._
 
+## Brush Anti-aliasing [GTK/WPF]
+
+On Skia targets (GTK and WPF), anti-aliasing is disabled by default for brushes, you can request it to be anti-aliased by setting this in your application's constructor:
+
+```csharp
+Uno.CompositionConfiguration.Configuration |= Uno.CompositionConfiguration.Options.UseBrushAntialiasing;
```

Comment:
I would wrap both of these samples in `#if HAS_UNO` as the API is not available on Windows